lib/theme/_colors.js

```diff
@@ -0,0 +1,10 @@
+export const colors = {
```

Comment:
dobra praktyka, to takie enumy raczej: 

export const Colors = {
  BEIGE: '#coasdlak"
  //....
}

plus raczej nie mieszałbym że coś jest kolorem bg - jeśli zostanie tak użyty to ok, natomiast ta tablica nie powinna zawierać takiej informacji

Reply:
Resolved